modules/installation-about-restricted-network.adoc

```diff
@@ -16,6 +17,9 @@ endif::[]
 ifeval::["{context}" == "installing-openstack-installer-restricted"]
 :osp:
 endif::[]
+ifeval::["{context}" == "installing-rhv-restricted-network"]
+:rhv-restricted:
```

Comment:
Since this is a module on restricted networks, I'd suggest following the other providers and just use the `:rhv:` attribute.